report_pre_thesis/src/index.tex

```diff
@@ -84,15 +86,7 @@ \section{Definition}
 \end{definition}
 
 \begin{definition}
-    A prediction set $\mathbf{\hat{y}}$ is the weigthed average of the predictions from the PLS and ICA models defined by:
-
-    \( \mathbf{\hat{y}} = w_{PLS} \times Y_{PLS} + w_{ICA} \times Y_{ICA} \)
-
-    Let:
-    \begin{itemize}
-        \item $Y_{PLS}$ and $Y_{ICA}$ be the prediction sets from the PLS and ICA models, respectively.
-        \item $w_{PLS}$ and $w_{ICA}$ be the weights assigned to the PLS and ICA models, respectively, such that \(w_{PLS} + w_{ICA} = 1 \), ensuring a weighted average.
-    \end{itemize}
+    Let \(h: S \rightarrow \mathbb{R}^m \) be a hypothesis function, where for each spectrum \(s \in S\), $h(s)$ produces a prediction \(\mathbf{\hat{y}} \in \mathbb{R}^m \). Here, $m$ denotes the dimensions of the composition predictions, and the structure and content of $\mathbf{\hat{y}}$ are analogous to a composition $k$. The function $h$ is determined by the model's learned parameters and the hyperparameters of the algorithm.
```

Comment:
- Remember that the (current) models do not take raw spectra.
- We do not define the units, merely that they are real numbers
- "_Here, $m$ denotes the dimensions of the composition predictions_" seems unclear
- Don't think we can just say 'seems analogous'
- My remark on "_The function $h$ is determined by the model's learned parameters and the hyperparameters of the algorithm._" was just for clarification between us - it's hopefully implicitly understood by anyone reading, as it is common knowledge in ML research.